packages/react-theme/src/utils/themeToCSSVariables.ts

```diff
@@ -0,0 +1,25 @@
+import { Theme } from '../types';
+
+function flattenThemeToCSSVariables(
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  themePart: Record<string, any>,
+  parentKey: string = '',
+  result: Record<string, string> = {},
+) {
+  for (const propertyName in themePart) {
+    if (Object.prototype.hasOwnProperty.call(themePart, propertyName)) {
```

Comment:
is there any harm in just calling `themePart.hasOwnProperty(propertyName)` directly?